tests/e2e/tests/build/with-post-css-warnings.ts

```diff
@@ -0,0 +1,43 @@
+import {
+  killAllProcesses,
+  waitForAnyProcessOutputToMatch,
+  execAndWaitForOutputToMatch
+} from '../../utils/process';
+import { appendToFile } from '../../utils/fs';
+import { getGlobalVariable } from '../../utils/env';
+import { updateJsonFile } from '../../utils/project';
+
+const webpackGoodRegEx = /webpack: Compiled successfully./;
```

Comment:
rename to Webpack Success

Reply:
I will do this, but it won't fix failing tests right?